app/javascript/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchLoadedData } from './components/redux/api';
+
+let isFetching = true;
+
+function App() {
+  const dispatch = useDispatch();
+
+  if (isFetching === true) {
+    isFetching = false;
+    dispatch(fetchLoadedData());
+  }
+
+  const result = useSelector((state) => state.data);
+  return (
+    <div className="App container">
+      <h2>Greetings from Api</h2>
+      {result.map((item) => (
+        <p key={item.id}>
+          {item.greeting}
+        </p>
+      ))}
+      <button type="button" onClick={() => { dispatch(fetchLoadedData()); }}>Greet</button>
+    </div>
+  );
```

Comment:
Kindly create a `Greeting` component that renders a single greeting.

- The App component should have a route that leads to the Greeting component as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/206799305-dc4e0b20-b1d7-4c11-9f40-01832b3230e2.png)

Kindly refactor your code so that the functionality for displaying a greeting is in the `Greeting` component.